src/MusicDrawer/styled-components.js

```diff
@@ -133,7 +131,6 @@ export const NowPlayingImage = styled.img.attrs({
   transform-origin: 0 0;
   display: block;
   margin-right: 1rem;
-  position: relative;
```

Comment:
`position` style duplicate with **line 139**, and I tested, `absolute` is right for `NowPlayingImage`